groups.go

```diff
@@ -97,10 +113,10 @@ func (s *GroupsService) GetGroup(gid interface{}, options ...OptionFunc) (*Group
 //
 // GitLab API docs: https://docs.gitlab.com/ce/api/groups.html#new-group
 type CreateGroupOptions struct {
-	Name            *string               `url:"name,omitempty" json:"name,omitempty"`
-	Path            *string               `url:"path,omitempty" json:"path,omitempty"`
-	Description     *string               `url:"description,omitempty" json:"description,omitempty"`
-	VisibilityLevel *VisibilityLevelValue `url:"visibility_level" json:"visibility_level,omitempty"`
+	Name                 *string                    `url:"name,omitempty" json:"name,omitempty"`
+	Path                 *string                    `url:"path,omitempty" json:"path,omitempty"`
+	Description          *string                    `url:"description,omitempty" json:"description,omitempty"`
+	GroupVisibilityLevel *GroupVisibilityLevelValue `url:"visibility" json:"visibility,omitempty"`
```

Comment:
@nilsjung all the changes in this file are not needed. Instead the currently existing `VisibilityLevelValue` type and constants should be updates to these new string values. See this item from the V3 to V4 docs:

![image](https://cloud.githubusercontent.com/assets/4171547/25055999/7bcd3b24-2166-11e7-9819-9f4b7f28e4ed.png)